rackcity/views/asset_views.py

```diff
@@ -193,25 +193,33 @@ def asset_add(request):
         validate_user_asset_permission_to_add(request.user, serializer.validated_data)
     except UserAssetPermissionException as auth_error:
         return JsonResponse(
-            {"failure_message": Status.AUTH_ERROR + str(auth_error)},
+            {"failure_message": Status.AUTH_ERROR.value + str(auth_error)},
             status=HTTPStatus.UNAUTHORIZED,
         )
     except Exception as error:
+        print("this error")
```

Comment:
can u remove this print statement  - CLAIRE LINTER

Reply:
Damn it
Claire linter - 1
Ben linter - 1